packages/puppeteer-core/src/puppeteer-core.ts

```diff
@@ -14,21 +14,16 @@
  * limitations under the License.
  */
 
-import {initializePuppeteer} from './initializePuppeteer.js';
-
-export * from './common/NetworkConditions.js';
-export * from './common/QueryHandler.js';
+export {Protocol} from 'devtools-protocol';
 export * from './common/DeviceDescriptors.js';
 export * from './common/Errors.js';
+export * from './common/NetworkConditions.js';
+export * from './common/QueryHandler.js';
+
+import {Puppeteer} from './common/Puppeteer.js';
 
-const puppeteer = initializePuppeteer('puppeteer-core');
+const puppeteer = new Puppeteer({isPuppeteerCore: true});
 
-export const {
-  connect,
-  createBrowserFetcher,
-  defaultArgs,
-  executablePath,
-  launch,
-} = puppeteer;
+export const {connect} = puppeteer;
```

Comment:
I think `launch` and other methods should be available still? I think it's perfectly fine to launch a browser with the user provided path using puppeteer-core. Other methods also make sense.